### DISCUSSION TITLE
Repositório  Release v4.0.56

### DISCUSSION THREAD

**Discussion by rrodrigomarinho** [wmixvideo/nfe#983] [Geral]
Onde encontro o repositório da Release v4.0.56?
A mesma não se encontra  no mavem

**Comment by edudoda** [comment]
Pois é @rrodrigomarinho  também não sei dizer porque não estão mais indo para o Maven... estou baixando o JAR e incluindo diretamente no POM, ex::

```
<dependency> 
	<groupId>com.github.wmixvideo</groupId>
						<artifactId>nfe</artifactId>
						 <version>4.0.57</version>  
						  <scope>system</scope>
					        <systemPath>/Users/xpto/.../lib/nfe-4.0.57-SNAPSHOT.jar</systemPath> 
			  </dependency>  
```

**Comment by fincatto** [comment]
Estamos com problemas no script automatizado que envia. O servidor do repositorio mudou recentemente e ainda não tivemos tempo de adaptar as novas mudanças, mas creio que, em breve, conseguiremos ajustar isso.
Se alguem tiver experiencia com isso e puder ajudar, só avisar o que precisa. 
Att

**Comment by rafaelstelles** [comment]
Eu tentei achar o erro e eu não vi @fincatto . Você está trabalhando esses ajustes em alguma branch ? Posso tentar dar uma olhada nisso...

**Comment by fincatto** [comment]
Está no master mesmo. Fiz algumas tentativas de liberação, sem sucesso.
--
Diego Fincatto
https://www.linkedin.com/in/fincatto/


Em sex., 12 de jul. de 2024 às 13:09, Rafael Sanches Telles <
***@***.***> escreveu:

**Reply by rafaelstelles** [reply]
Pelo erro abaixo, parece ser algo relacionado a autenticacão.

`Failed to execute goal org.sonatype.plugins:nexus-staging-maven-plugin:1.7.0:deploy (injected-nexus-deploy) on project nfe: Execution injected-nexus-deploy of goal org.sonatype.plugins:nexus-staging-maven-plugin:1.7.0:deploy failed: 400 - Bad Request -> [Help 1]`

Você tentou rodar no modo debug para entender o que ele está passando para frente de informação (url/user/pass) ?

Eu achei [esse](https://stackoverflow.com/questions/58214572/is-there-anyway-to-sign-maven-package-at-github-action-workflows) link também, que também parece fazer sentido a sugestão da última solução, para não a action `samuelmeuli/action-maven-publish@v1.4.0` que é antiga e não tem mais suporte.

De forma geral, parece que realmente o problema está em credenciais, porém como eu não tenho acesso pra conferir as informações e entender até onde ele está indo, fica díficil de ajudar nesse caso.

**Comment by fincatto** [comment]
Oi pessoal, bom dia. 
O problema foi uma mudança no OSSRH, que usamos para hospedar o projeto. 
Já ajustamos e conseguimos fazer o upload pra eles ontem ainda, embora não esteja aparecendo no maven central. 
Estamos investigando aqui, mas cremos que em breve vai estar sincronziando direitinho.

**Comment by fincatto** [comment]
Tá feito! 
https://central.sonatype.com/namespace/com.github.wmixvideo
